services/src/datasets/storage.rs

```diff
@@ -170,16 +171,17 @@ pub enum MetaDataDefinition {
     OgrMetaData(StaticMetaData<OgrSourceDataset, VectorResultDescriptor, VectorQueryRectangle>),
     GdalMetaDataRegular(GdalMetaDataRegular),
     GdalStatic(GdalMetaDataStatic),
+    GdalMetadataNetCdfCf(GdalMetadataNetCdfCf),
 }
 
 impl MetaDataDefinition {
     pub fn source_operator_type(&self) -> &str {
         match self {
             MetaDataDefinition::MockMetaData(_) => "MockDatasetDataSource",
             MetaDataDefinition::OgrMetaData(_) => "OgrSource",
-            MetaDataDefinition::GdalMetaDataRegular(_) | MetaDataDefinition::GdalStatic(_) => {
-                "GdalSource"
-            }
+            MetaDataDefinition::GdalMetaDataRegular(_)
```

Comment:
we should wrap them into a new ENUM and  move that to the  gdal_source folder.

Reply:
s.o.

additionally, keep in mind that this is a different crate